test/parallel/test-http-chunked-304.js

```diff
@@ -12,32 +12,35 @@ var net = require('net');
 // Likewise for 304 responses. Verify that no empty chunk is sent when
 // the user explicitly sets a Transfer-Encoding header.
 
-test(204, function() {
-  test(304);
-});
+test(204);
+test(304);
 
-function test(statusCode, next) {
-  var server = http.createServer(function(req, res) {
+function test(statusCode) {
+  const server = http.createServer(common.mustCall((req, res) => {
     res.writeHead(statusCode, { 'Transfer-Encoding': 'chunked' });
     res.end();
     server.close();
-  });
+  }));
 
-  server.listen(0, function() {
-    var conn = net.createConnection(this.address().port, function() {
-      conn.write('GET / HTTP/1.1\r\n\r\n');
+  server.listen(0, common.mustCall(() => {
+    const conn = net.createConnection(
```

Comment:
@lpinca didn't get an error before with `make -j8 lint`, but changed it this way to make it look better, same.. no error with `make -j8 lint`, let me know if this way works or if not please provide an example and will make it fit